legacy/src/app/routes.js

```diff
@@ -227,6 +233,26 @@ const configureRoutes = ($stateProvider, $urlRouterProvider) => {
       template: dashboardTmpl,
       controller: "DashboardController",
     });
+
+  $urlRouterProvider.otherwise(($injector, $location) => {
```

Comment:
unfortunately `$rootScope` isn't available in the angularjs `config` function, so I can't reuse caleb's nice navigation functions.